homeassistant/components/brother/sensor.py

```diff
@@ -43,36 +52,42 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 class BrotherPrinterSensor(CoordinatorEntity, SensorEntity):
     """Define an Brother Printer sensor."""
 
-    def __init__(self, coordinator, kind, device_info):
+    def __init__(
+        self,
+        coordinator: BrotherDataUpdateCoordinator,
+        kind: str,
+        device_info: dict[str, Any],
+    ) -> None:
         """Initialize."""
         super().__init__(coordinator)
+        assert coordinator.data is not None
         self._name = f"{coordinator.data.model} {SENSOR_TYPES[kind][ATTR_LABEL]}"
```

Comment:
Then save `SensorDescription` right here to not check if `self.kind in SENSOR_TYPES` every time.

Reply:
Sorry, I don't understand.